assignments/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783/assign.tagging.json

```diff
@@ -0,0 +1,26 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tags",
+      "displayName": "HMCTS Tagging - CFT SBOX (DCD-CFT-Sandbox)",
+      "enforcementMode": "DoNotEnforce",
+      "nonComplianceMessages": [
+        {
+            "message": "Resource doesn't meet the tagging requirements. See https://github.com/hmcts/azure-policy/blob/master/policies/tagging/README.md"
+        }
+      ],
+      "notScopes": [],
```

Comment:
aks node rg not found under non compliant list for that subscription, therefore no scope was excluded.